controllers/topolvm_node.go

```diff
@@ -152,27 +190,25 @@ func getNodeDaemonSet(lvmCluster *lvmv1alpha1.LVMCluster) *v1.DaemonSet {
 	if tolerations != nil {
 		topolvmNodeTolerations = tolerations
 	}
-
+	labels := map[string]string{
+		"app":                   topolvmNodeName,
+		"topolvm.io/lvmcluster": lvmCluster.Name,
```

Comment:
- there's a possibility that this might become ambiguous as there are recent discussions to supports multiple LVM CRs
- let's consider three nodes, out of which create daemonset targeted a single node initially and later another CR targeted remaining nodes, in that scenario this label will represent new CR only
- unless there's specific use IMHO it's better not to use this specific label

Reply:
In the case you expose, what we will  have is a couple of CRs with the same labels. Is clear that, we will need to implement the management of several CRs, but i cannot see the problem of having the same labels in two CRs. Can you remark what is the problem you see?

Reply:
Ack, I'm concerned about `topolvm.io/lvmcluster` value as this'll be overwritten in all node daemonsets if there's a new CR.

Reply:
We don't need this label.

Reply:
I will remove